.buildkite/scripts/common.sh

```diff
@@ -90,11 +90,6 @@ retry() {
     return 0
 }
 
-docker_logout() {
```

Comment:
the BK plugin is the one doing the logout now:

<img width="773" alt="image" src="https://github.com/user-attachments/assets/ce6506e0-0a94-4ff6-8109-9bdbea6849a4" />